tests/ui/impl-trait/in-trait/refine-captures.stderr

```diff
@@ -37,9 +35,8 @@ LL |     fn test() -> impl Sized + use<'b> {}
    = note: we are soliciting feedback, see issue #121718 <https://github.com/rust-lang/rust/issues/121718> for more information
 help: modify the `use<..>` bound to capture the same lifetimes that the trait does
    |
-LL -     fn test() -> impl Sized + use<'b> {}
-LL +     fn test() -> impl Sized + use<'a, 'b> {}
-   |
+LL |     fn test() -> impl Sized + use<'a, 'b> {}
+   |                                    ++++
```

Comment:
LOL @ the suggestion being `"a, '"` rather than `"'a, "`. I don't think this is worth tweaking tho, if the suggestion was smarter then it wouldn't be overwriting the whole `+ use<>` in the first place.

Reply:
Hey! diffing algorithms producing what you expect is always hard ^_^

And this is an improvement nonetheless.